app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,28 @@
+class StaticPagesController < ApplicationController
+  before_action :set_locale
+
+  def home; end
+
+  def help; end
+
+  def contact; end
+
+  def about; end
+
+  private
+
+  def default_url_options
+  {  locale: I18n.locale}
+  end
+
+  def set_locale
+    I18n.locale = extract_locale || I18n.default_locale
+  end
+
+  def extract_locale
+    parsed_locale = params[:locale]
+    I18n.available_locales.map(&:to_s).include?(parsed_locale) ?
+    parsed_locale.to_sym :
+    nil
```

Comment:
indent line 24 - 26 sai sai em ơi?
giải thích line code 24 - 26 luôn giúp chị

Reply:
Dạ nó sẽ kiểm tra cái đuôi của url xem thử nó có nằm trong với ngôn ngữ được hỗ trợ hay không nếu có thì sẽ in ra còn không sẽ trả về nil

Reply:
ready